microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,184 @@ ms.date: 05/04/2021
+
+To learn how to show the network activity that's generated by the page resources, see [Log network activity](../network/index.md#log-network-activity) in the article  _Inspect network activity_.
 
-### Reveal resources in the Network tool from other panels
 
-The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
 
-![Revealing resources in the Network panel.](../media/resources-sources-page-reveal-in-network-panel.msft.png)
 
 
 <!-- ====================================================================== -->
-## Browse resources in the Network panel
+## Show resource files in the Network tool from other tools
+
+In various tools that list resource files for the webpage, to inspect a resource file in the **Network** tool, right-click the resource file and then select **Reveal in Network panel**.  You may need to first refresh the webpage while the **Network** tool is open.
+
+For example, to go from the **Page** tab's resource list in the **Sources** tool to the **Network** tool:
+
+1. In a new window or tab, go to a webpage, such as the [Inspect Network Activity Demo](https://microsoftedge.github.io/Demos/network-tutorial/).  (The source files for that webpage are at [MicrosoftEdge / Demos > network-tutorial](https://github.com/MicrosoftEdge/Demos/tree/main/network-tutorial).)
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-<!-- to do x, -->
+1. In DevTools, on the main toolbar, select the **Network** tab.  If that tab isn't visible, click the **More tabs** (![the More tabs button](../media/more-tabs-icon-light-theme.png)) button, or else the **More Tools** (![More Tools icon](../media/more-tools-icon-light-theme.png)) button.
 
-1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window.
+1. Press `Ctrl`+`R` to refresh the page.  The received resource files that make up the webpage are now added as rows in the **Network** tab.  The **demo-to-do/** folder row represents the HTML page - in this case, [index.html](https://github.com/MicrosoftEdge/Demos/tree/main/demo-to-do).
 
-<!-- 1. Do the steps in that section, and then continue below. -->
+1. In DevTools, click the **Sources** tab.
 
- <!-- open [link] in a new tab or window, and then continue below.. -->
-<!--why?-->
+1. In the **Page** tab in the upper left, right-click a resource file, and then select **Reveal in Network panel**:
 
-<!-- need lead-in -->
+   ![The 'Reveal in Network panel' right-click menuitem on a resource file in the Page tab of the Sources tool](../media/resources-sources-page-reveal-in-network-panel.msft.png)
```

Comment:
3 - Redo screenshot with the default DevTools docking location, to the right not bottom.  Adjust width of webpage and DevTools so window is narrow yet still recognizable.
Add red boxes on:
* The **Sources** tab.
* The **Page** tab.
* The **Reveal in Network panel** menuitem.

Reply:
done